File: historyscan-back/src/main/java/io/sch/historyscan/HistoryscanCommandLineRunner.java
package io.sch.historyscan;

import org.springframework.boot.CommandLineRunner;
import org.springframework.context.annotation.Profile;
import org.springframework.stereotype.Component;

@Component
@Profile("interactive")
public class HistoryscanCommandLineRunner implements CommandLineRunner {
    @Override
    public void run(String... args) {
        throw new UnsupportedOperationException("Not implemented yet");
    }
}

File: historyscan-back/src/main/kotlin/io/sch/historyscan/HistoryscanCommandLineRunner.kt
package io.sch.historyscan

import org.springframework.boot.CommandLineRunner
import org.springframework.context.annotation.Profile
import org.springframework.stereotype.Component

@Component
@Profile("cmd-line")
class HistoryscanCommandLineRunner : CommandLineRunner {
    override fun run(vararg args: String) {
        throw UnsupportedOperationException("Not implemented yet")
    }
}
